src/aiokem/main.py

```diff
@@ -38,64 +50,51 @@ def __init__(self, session: ClientSession = None) -> None:
         """Initialize the AioKem class."""
         self._token: str | None = None
         self._refresh_token: str | None = None
-        self._session = (
-            session if session else ClientSession(timeout=ClientTimeout(total=10))
-        )
-        _LOGGER.info("AioKem instance initialized.")
+        self._session = session or ClientSession(timeout=CLIENT_TIMEOUT)
 
-    async def _authentication_helper(
-        self, headers: CIMultiDict, data: CIMultiDict
-    ) -> None:
+    async def _authentication_helper(self, data: dict[str, Any]) -> None:
+        _LOGGER.debug("Sending authentication request to %s", AUTHENTICATION_URL)
         try:
-            _LOGGER.debug("Sending authentication request to %s", AUTHENTICATION_URL)
             response = await self._session.post(
-                AUTHENTICATION_URL, headers=headers, data=data
+                AUTHENTICATION_URL, headers=AUTH_HEADERS, data=data
             )
             response_data = await response.json()
-            if response.status != HTTPStatus.OK:
```

Comment:
We should move code out of try blocks that can't raise what the except is for

Reply:
yes, that's a good point